app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/topics-controllers.js");
+
+app.use(express.json());
```

Comment:
this code isn't being used here (because it's only needed for posts etc) so therefore is just boilerplate, rather than efficient use of code. I would delete it as it technically isn't tested for! :)